lib/std/zig/system/NativePaths.zig

```diff
@@ -112,6 +112,19 @@ pub fn detect(arena: Allocator, native_target: std.Target) !NativePaths {
         return self;
     }
 
+    if (builtin.os.tag == .netbsd) {
+        // find local built libs (like LLVM) and pkgin installed libs
+        try self.addIncludeDir("/usr/local/include");
+        try self.addIncludeDir("/usr/pkg/include");
+        try self.addIncludeDir("/usr/include");
+
+        try self.addLibDir("/usr/local/lib");
+        try self.addLibDir("/usr/pkg/lib");
+        try self.addLibDir("/usr/lib");
+        try self.addLibDir("/lib");
+        return self;
+    }
+
```

Comment:
While this may be the correct thing to do for netbsd's packaging system and ports, generally we do not add more native paths than what the system compiler has.

Secondly, we do not need to explicitly add `/usr/include` and `/usr/lib` because an on-demand method is used. `zig libc` command can be used to see what is determined. Also `zig cc -c foo.c -v` shows default includes or to see linking and lib dirs `zig cc -o foo foo.c -v`.

Thus this change is rejected.